java/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java

```diff
@@ -209,6 +210,24 @@ public static void createApplication(String appName, String appOwner, String app
             localAndOutboundAuthenticationConfig.setAuthenticationType("flow");
             localAndOutboundAuthenticationConfig
                     .setAuthenticationSteps(new AuthenticationStep[]{authenticationStep1, authenticationStep2});
+
+            // Need to enable username validation by passing the adaptive parameter.
+            String authenticationScript = "var onLoginRequest = function(context) {\n" +
+                "    executeStep(1, {\n" +
+                "        authenticatorParams: {\n" +
+                "            common: {\n" +
+                "                \"ValidateUsername\": \"true\"\n" +
+                "            }\n" +
+                "        }\n" +
+                "    }, {\n" +
+                "        onSuccess: function(context) {\n" +
+                "            executeStep(2);\n" +
+                "        }\n" +
+                "    });\n" +
+                "};";
+            AuthenticationScriptConfig authenticationScriptConfig = new AuthenticationScriptConfig();
+            authenticationScriptConfig.setContent(authenticationScript);
```

Comment:
Shall we check whether we need to enable the script
authenticationScriptConfig.enable or something